package.json

```diff
@@ -1,10 +1,10 @@
 {
     "dependencies": {
         "browser-resolve": "^1.11.2",
-        "create-react-class": "^15.5.3",
+        "create-react-class": "^15.6.0",
         "module-deps": "^4.1.1",
-        "react": "^15.5.3",
-        "react-dom": "^15.5.3",
+        "react": "^16.0.0",
+        "react-dom": "^16.0.0",
```

Comment:
why not strict?

Reply:
As these have always been like this, and I don't know much about Node best practices.

So it would be best to not usre any specifiers, ~ or ^ etc.?

Reply:
Well, on the `:npm-deps` you have them strict, so at least for consistency.
I think it would be better to have a bit more determinism by omitting the `^`

Reply:
best to not use any specifiers imo. that way one can test against a known good configuration and know that users will get that same configuration.